api.md

```diff
@@ -908,7 +908,8 @@ SELECT time_bucket('5 minutes', time, '-2.5 minutes') + '2.5 minutes'
   ORDER BY five_min DESC LIMIT 10;
 ```
 
-To shift the alignment of the buckets you can use the origin parameter.
+To shift the alignment of the buckets you can use the origin parameter
+(passed as a timestamp, timestamptz, or date type).
```

Comment:
Our more common style is to use all uppercase for types. There are a few exceptions in this page, so maybe you can change those too.

Reply:
Actually, if those other changes apply to older versions than 1.0, it would be better to put in a separate PR.